src/transformStateWithClones.js

```diff
@@ -66,6 +66,36 @@
  */
 function transformStateWithClones(state, transforms) {
   // write code here
+  const result = [];
+  let clone = Object.assign({}, state);
+
+  for (const action of transforms) {
+    switch (action.operation) {
+      case 'addProperties': {
+        result.push(Object.assign({}, clone,
+          action.properties));
+        break;
```

Comment:
так дужки не переносять 
```suggestion
        result.push(Object.assign(
          {},
          clone,
          action.properties
        ));
        break;
```